src/nodeBuffer.ml

```diff
@@ -108,151 +108,151 @@ external fillString : t -> Js.String.t -> t = "fill" [@@bs.send]
 external fillStringOffset : t -> value:Js.String.t -> offset:int -> t =
   "fill" [@@bs.send]
 external fillStringRange : t -> value:Js.String.t -> offset:int ->
-  offsetEnd:int -> t = "fill" [@@bs.send]
+  end_:int -> t = "fill" [@@bs.send]
 external fillStringRangeWithEncoding : t -> value:Js.String.t -> offset:int ->
-  offsetEnd:int -> encoding:Js.String.t -> t = "fill" [@@bs.send]
+  end_:int -> encoding:Js.String.t -> t = "fill" [@@bs.send]
 external fillBuffer : t -> t -> t = "fill" [@@bs.send]
 external fillBufferOffset : t -> value:t -> offset:int -> t = "fill" [@@bs.send]
-external fillBufferRange : t -> value:t -> offset:int -> offsetEnd:int -> t =
+external fillBufferRange : t -> value:t -> offset:int -> end_:int -> t =
   "fill" [@@bs.send]
 external fillInt : t -> int -> t = "fill" [@@bs.send]
 external fillIntOffset : t -> value:int -> offset:int -> t = "fill" [@@bs.send]
-external fillIntRange : t -> value:int -> offset:int -> offsetEnd:int -> t =
+external fillIntRange : t -> value:int -> offset:int -> end_:int -> t =
   "fill" [@@bs.send]
 
 external includesString : t -> Js.String.t -> bool = "includes" [@@bs.send]
-external includesStringOffset : t -> value:Js.String.t -> offset:int -> bool =
+external includesStringFrom : t -> value:Js.String.t -> offset:int -> bool =
   "includes" [@@bs.send]
-external includesStringOffsetWithEncoding : t -> value:Js.String.t -> offset:int ->
+external includesStringFromWithEncoding : t -> value:Js.String.t -> offset:int ->
```

Comment:
It sounds better but don't match arguments order. On the other side, they are labeled anyway, so let's try your option.